docs/Testing/OSXWinObjCTests.md

```diff
@@ -0,0 +1,172 @@
+# WinObjC Unit Tests on OSX
+
+As a measure to help ensure the accuracy of the WinObjC platform, we are beginning to support building and running WinObjC unit tests on OSX devices. By verifying our tests against the reference platform, we hope to reduce errors in test writing and incorrect assumptions about Objective-C APIs. Foundation.UnitTests are currently supported, with other test modules expected to follow suit by the end of July.
```

Comment:
> by the end of July. [](start = 389, length = 20)

perhaps remove time specific wording since this could change and the doc will be there for "all time"